netket/experimental/sampler/metropolis_pt.py

```diff
@@ -129,8 +129,22 @@ class MetropolisPtSampler(MetropolisSampler):
 
     The total number of chains evolved is :code:`n_chains * n_replicas`.
     """
+    _beta_sorted: jax.Array = None
+    """
+    An internal cache for the user-specified betas, sorted.
+    """
+    _beta_distribution: str = struct.field(pytree_node=False, default="lin")
+    """
+    An internal for the user-specified distribution of betas.
+    """
 
-    def __init__(self, *args, n_replicas: int = 32, **kwargs):
+    def __init__(
+        self,
+        *args,
+        n_replicas: Optional[int] = None,
+        betas: Optional[Union[str, jax.Array]] = None,
```

Comment:
The default can be specified here. Leads to more clear docstrings and a slightly simpler implementation.
```suggestion
        betas: Optional[Union[str, jax.Array]] = "linear",
```